android/java/brave-res/xml/brave_main_preferences.xml

```diff
@@ -5,16 +5,49 @@
      You can obtain one at https://mozilla.org/MPL/2.0/. -->
 
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:orderingFromXml="false">
     <PreferenceCategory
         android:key="brave_account_section"
         android:order="1"
-        android:title="@string/prefs_section_brave_account"/>
+        android:title="@string/prefs_brave_account"/>
```

Comment:
I would do `app:isPreferenceVisible="false"` to avoid a brief flash before removal when the feature is disabled.

Reply:
discussed that in the DM and it's not visible and fine to leave as is as we are going to enable the feature at the end.

Reply:
We talked this through on Slack and agreed not to use `app:isPreferenceVisible="false"`, since there are no visual artifacts when Settings loads with Brave Account disabled.